packages/lib/src/utils/Logger.ts

```diff
@@ -132,14 +134,27 @@ export const Loggy: { [key: string]: any } = {
     }
   },
 
+  _actionToText(action: SpinnerAction): string {
+    switch (action) {
+      case SpinnerAction.Add || SpinnerAction.NonSpinnable:
+        return 'added';
```

Comment:
I'd change this to 'started' (I cannot add suggestions reviewing commit-by-commit)